src/index.html

```diff
@@ -12,17 +12,17 @@
     <h1>Daily Prebuilt Kahoot Integration Demo</h1>
     <div class="welcome-screen" id="welcomeScreen">
       <p>
-        Start a Kahoot game from the apps listed in the embedded video chat
-        widget. Add your Daily room URL to script.js and click 'Join room'.
+        Start a Kahoot game using the 'Start Kahoot Host' button in the call screen. 
```

Comment:
```suggestion
        Start a Kahoot game using the 'Start Kahoot' button in the call screen. 
```

I think the button does not actually have the word "Host" _in_ it